Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -11973,6 +11983,10 @@ a_Player:OpenWindow(Window);
 					},
 					Notes = "Sets if the player can fly or not.",
 				},
+				SetCapabilities =
+				{
+					Notes = "Updates player's capabilities - flying, visibility, etc. from their gamemode.",
+				},
```

Comment:
I checked the code and I see that the function is only called when the game mode changes. Remove from export?